File: app/src/main/java/com/example/leeeyou/zhihuribao/data/model/RiBao.java
package com.example.leeeyou.zhihuribao.data.model;

import com.google.gson.Gson;

import java.util.List;

/**
 * Created by leeeyou on 16/5/10.
 * <p/>
 * getLatestRiBao返回的数据结构
 */
public class RiBao {
    public String date;
    public List<Story2> stories;

    @Override
    public String toString() {
        return new Gson().toJson(this);
    }
}

File: app/src/main/java/com/xyz/leeeyou/zhihuribao/data/model/ribao/RiBao.kt
package com.xyz.leeeyou.zhihuribao.data.model.ribao

/**
 * ClassName:
 * Description:  The data class is a very powerful class that
 *               lets you avoid creating boilerplate code in Java
 *               that holds POJOs that are state-controlled but have a very simple operation.
 *
 *               They usually only provide simple getters and setters for accessing their properties.
 *
 * Author:      leeeyou
 * Date:        2017/4/21 15:07
 */
data class RiBao(var date: String, var stories: List<Story>)